CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,118 @@
+  * Violent language or threats or personal insults have no chance to
+    resolve a dispute or to let a discussion florish. Worse, they can
+    hurt durably, or generate durable fears. They are thus unwelcome.
+  * Not everyone is comfortable with sexually explicit or violent
+    material, even as a joke. In an online open multicultural world, you
+    don't know who might be listening. So be cautious and responsible
+    with your words.
+  * Discussions are online and recorded for posterity; we all have our
+    right for privacy and online gossiping as well as posting or threatening to
+    post other people's personally identifying information is prohibited.
+- **Remember that what you write in a public online forum might be read by
+  many people you don't know.**
+  Consider what image your words will give to outsiders of the development
+  team / the user community as a whole. Try to avoid references to private
+  knowledge to be understandable by anyone.
+- **Coq online forums are only to discuss Coq-related subjects.**
+  Unrelated political discussions or long digressions are unwelcome,
+  even for illustration or comparison purposes.
+- **When we disagree, try to understand why.**
+  Disagreements, both social and technical, happen all the time and Coq is no
+  exception. It is important that we resolve disagreements and differing views
+  constructively. Remember that we are different. Different people
+  have different perspectives on issues. Being unable to understand why someone
+  holds a viewpoint doesn't mean that they're wrong.
+- **It is human to make errors, and please try not to take things personally.**
+  Please do not to answer aggressively to problematic behavior and simply
+  signal the issue. If actions have been taken with you (e.g. bans or simple
+  demands of apology, of rephrasing or keeping personal beliefs or troubles
+  private), please understand that they are not intended as aggression or
+  punishment ― even if you they feel harsh to you ― but as ways to enforce a
+  calm communication for the other participants and to give you the opportunity
+  to change your behavior. We understand you may feel hurt, or maybe you had a
+  bad day, so please take this opportunity to question yourself, cool down if
+  necessary and do not persist in the exact same behavior you have been
+  reported for.
+
+## Enforcement ##
+
+If you believe someone is violating the code of conduct, we ask that you report
+it by emailing the Coq Code of Conduct enforcement team at
+<coq-conduct@inria.fr>. Confidentiality with regard to the reporter of an
+incident will be maintained while dealing with it.
+
+In particular, you should seek support from the team instead of dealing by
+yourself with a behavior that you consider hurtful. This applies to members of
+the enforcement team as well, who shouldn't deal by themselves with violations
+in discussions to which they are a participant.
```

Comment:
in discussions in which they are a participant (alternatively: in discussions to which they participate)

Reply:
> in discussions to which they participate

I don't think this one works.  I think "in discussions to which they are party" is at least correct grammar, even if it's a bit awkward.  (The rule I'm using: "they participate to discussions" is not valid, but "they are party to discussions" is.)

You could also use "in discussions in which they participate"

Reply:
So @JasonGross, which one is better?

Reply:
I'd say just change "to which" to "in which", as @ybertot suggested before his parenthetical.